crates/holochain/src/sweettest/sweet_conductor.rs

```diff
@@ -712,11 +713,17 @@ impl SweetConductor {
         }
     }
 
-    /*
-    /// Let each conductor know about each others' agents so they can do networking
-    pub async fn exchange_peer_info(conductors: impl Clone + IntoIterator<Item = &Self>) {
-        let mut all = Vec::new();
-        for c in conductors.into_iter() {
+    /// Let each conductor know about each other's agents so they can do networking.
+    ///
+    /// Returns a boolean indicating whether each space has at least one agent info for each conductor.
```

Comment:
It seems the agent info might not always be available locally right away now. Make it possible to retry if we're not seeing the minimum number of agents we'd expect.